src/settings.py

```diff
@@ -261,8 +261,9 @@ class GlobalSettings:
     llm_openai_max_response_tokens = 250
 
     # parameters for open loop LLM approach
-    open_loop_llm_model_name = "text-curie-001"
-    open_loop_llm_temperature = 0.0
+    open_loop_llm_model_name = "text-davinci-002"
```

Comment:
is this change something you want to commit to master? @tomsilver

Reply:
i'll revert it to be safe